components/input-number/index.tsx

```diff
@@ -35,6 +35,10 @@ export default class InputNumber extends React.Component<InputNumberProps, any>
       [`${this.props.prefixCls}-sm`]: size === 'small',
     }, className);
 
-    return <RcInputNumber className={inputNumberClass} {...others} />;
+    return <RcInputNumber ref="inputnumber" className={inputNumberClass} {...others} />;
```

Comment:
Use callback for ref.

Reply:
We should trun on https://github.com/palantir/tslint-react/tree/master/test/rules/jsx-no-string-ref

Reply:
Thanks @yesmeck I adjusted accordingly